app/Services/CampaignService.php

```diff
@@ -89,6 +88,7 @@ public function getPendingPostTotals($campaigns)
                 ->leftJoin('posts', 'signups.id', '=', 'posts.signup_id')
                 ->selectRaw('signups.campaign_id, count(posts.id) as pending_count')
                 ->where('status', '=', 'pending')
+                ->whereIn('type', ['photo', 'text'])
```

Comment:
@DoSomething/team-bleed I think we also have to update here so that these numbers are the same on the `/campaigns` page and `/campaigns/:campaign_id`

Reply:
Can we update this query to run on the `posts` table as well (like we did for `getPostTotals` above)? I think that'd probably be another big performance win for this interface & should be a pretty straightforward change.

Reply:
ah yes I was going to do that but when I first looked thought we needed the join so didn't but realize now we totally don't! thanks @DFurnes , updated!